src/quote.c

```diff
@@ -34,7 +34,7 @@
  * Append a string to the buffer, and escape special characters.
  */
 static char *
-escape(char *dest, char *end, const char *value)
+escape(char *dest, const char *end, const char *value)
```

Comment:
While this pointer can technically be const, this would suggest that it points to a different memory area than `dest`, which is not const. This reduces readability.